velox/expression/PeeledEncoding.h

```diff
@@ -32,6 +32,11 @@ class LocalSelectivityVector;
 /// translate top level rows into inner rows and wrap a vector (typically
 /// generated as a result of applying an expression on the peeled vectors).
 ///
+/// When the base vector size is larger than 8 times of the selected rows, we do
+/// not save the dictionary wrapping.  Instead we store a flatten version of the
```

Comment:
nit: double spaces between sentences?
nit2: "Instead, we store a flattened version ..."

Reply:
Yeah prefer double space after sentence for monospace font.